pkg/sql/schemachanger/eav/eavquery/testdata/basic

```diff
@@ -0,0 +1,57 @@
+schema
+i32: Int32
+i64: Int64
+u32: Uint32
+s:   String
+----
+
+# Define 5 entities for use below. Note e5 and e1 are the same.
+entities
+e1: {i32: 1, i64: 1, u32: 1, s: hi}
+e2: {i32: 1, i64: 1, u32: 2, s: hi}
+e3: {i32: 1, i64: 1, u32: 1}
+e4: {i32: 1, i64: 1}
+e5: {i32: 1, i64: 1, u32: 1, s: 'hi'}
+e6: {i32: 2, i64: 1, u32: 1, s: 'hi'}
+e7: {i32: 2, i64: 1, u32: 2, s: yo}
+e8: {i32: 2, i64: 1, u32: 2, s: sup}
+e9: {i32: 3, i64: 2, u32: 2, s: yo}
+e10: {i32: 4, i64: 2, u32: 2, s: howdy}
+----
+
+query name=q
+- a[i32] = b[i32]
```

Comment:
Maybe it's just me but I definitely don't understand this query language. Is there a piece of documentation on it somewhere?